builtInFunctions/saveUserName.go

```diff
@@ -74,7 +75,7 @@ func inputCheckForUserNameCall(
 	if vmInput.CallValue.Cmp(zero) != 0 {
 		return ErrBuiltInFunctionCalledWithValue
 	}
-	if vmInput.GasProvided < gasCost {
+	if !check.IfNil(acntSnd) && vmInput.GasProvided < gasCost {
```

Comment:
[question]

This is related to the backwards compatibility flag: `s.isChangeEnabled()`, right?

Reply:
yes.